libavb_ab/avb_ab_flow.c

```diff
@@ -152,17 +152,12 @@ static void slot_normalize(AvbABSlotData* slot) {
       /* We've exhausted all tries -> unbootable. */
       slot_set_unbootable(slot);
     }
-    if (slot->tries_remaining > 0 && slot->successful_boot) {
```

Comment:
any reason to remove the code？

Reply:
These codes are from kernelflinger. Currently the avb code used by bootctrl have some differences from the code used by kernelflinger.

Jianfeng submitted a patch for remove this line, his commit message:

slot-ab: enable boot when tries > 1

    SetActiveBootSlot() of boot_control 1.1 always set slot
    tries_remaining to 6.
    Boot should be enabled in this case.